content/docs/cosidriver/_index.md

```diff
@@ -51,6 +51,8 @@ Dell COSI Driver is a multi-backend driver, meaning that it can connect to multi
 | ObjectScale      |  1.2.x   |
 {{</table>}}
 
+> **NOTE:** Object Scale 1.2.x is planned for End of Standard Support on January 31st 2025. Please refer <a href="https://www.dell.com/support/kbdoc/en-uk/000185734/all-dell-emc-end-of-life-documents?lang=en">Dell Support documentation</a> for more information. We plan to support COSI driver when a new version of ObjectScale is available.
```

Comment:
Is this "Dell End of Service Life (EOSL)" or "End of Life (EOL)"? If either use this nomenclature as opposed to "End of Standard Support" Thanks!

Reply:
Is it the EOSS Date as shown in the Notifications spreadsheet in the Dell Support page linked in the note.
![image](https://github.com/user-attachments/assets/e772f2ab-ca49-4dc9-a9af-57b05888d68a)
The spreadsheet elaborates that EOSS is "The date after which Standard Support is not available"

Reply:
EOL and EOSL dates were not published.

Reply:
Thanks for the clarification. All set from my side.